test/junit/src/test/scala/strawman/collection/SeqLikeTest.scala

```diff
@@ -23,4 +23,14 @@ class SeqLikeTest {
     assertEquals(List(List(1, 2), List(1, 3), List(2, 3)), List(1, 2, 3).combinations(2).toList)
     assertEquals(List(List(1, 2, 3)), List(1, 2, 3).combinations(3).toList)
   }
+  
+  @Test def intersperse(): Unit = {
+    import strawman.collection.immutable.List
```

Comment:
Is it okay to import our own List implementation here for now?

Reply:
Yes